src/configs/typescript.ts

```diff
@@ -0,0 +1,28 @@
+import type { Config } from '../types'
+
+const scripts: Config = {
+  scripts: [{
+    name: 'typecheck',
+    command: 'nuxt typecheck'
+  }, {
+    name: 'start',
+    command: 'NODE_ENV=production node .output/server/index.mjs'
```

Comment:
why is this command in the typescript config?

Reply:
Generally speaking it does not 100% relate to this, however as the project uses typescript by default, I added this here, to avoid having the create another config, just for this command.